six/modules/python/six/tests/utils.py

```diff
@@ -43,6 +43,13 @@ def findSixHome():
 
     return currentPath
 
+def sicdDir():
```

Comment:
This is accomplished.

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Tuesday, August 02, 2016 9:26 PM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>; Author <author@noreply.github.com>
Subject: Re: [ngageoint/six-library] Make crop_sicd's method of finding the appropriate directories more reliable. (#95)


In six/modules/python/six/tests/utils.py<https://github.com/ngageoint/six-library/pull/95#discussion_r73267481>:

> @@ -43,6 +43,13 @@ def findSixHome():

>

>      return currentPath

>

> +def sicdDir():

Can we put these directories in Jenkins rather than SIX?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ngageoint/six-library/pull/95/files/3e1c6db70430c655e910abcdf14206921627c7ed#r73267481>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFRAYAWYEI_iJ3o9gvV2127AgpLLE6Dzks5qb-4dgaJpZM4Jajll>.


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.